app/controllers/departments_controller.rb

```diff
@@ -1,8 +1,16 @@
 class DepartmentsController < ApplicationController
-  before_action :find_department, only: [:show, :edit, :update, :destroy]
+  before_action :find_department, only: [:show, :edit, :update, :destroy, :employees_list]
 
   def index
     @departments = Department.all
+
+    @department_manager = @departments.each_with_object({}) do |department, result|
```

Comment:
I would create a decorator for a department with the method `current_manager` and call the method on the view.